pkg/app/bp.go

```diff
@@ -44,6 +45,7 @@ type AppBlueprint struct {
 	Path           string
 	UseDevImages   bool
 	readFromBPRepo bool
+	ImageTag       string
```

Comment:
I was wondering if we need `UseDevImages` anymore. We can pass dev image tag as a ImageTag arg so I believe we can get rid of `UseDevImages`.

Reply:
What I am suggesting, we can get rid of `UseDevImages` since we have the ability pass `ImageTag`? For Kanister integration test, we can pass `ImageTag=v9.99.9-dev`